scripts/utils/sign_and_send.js

```diff
@@ -24,8 +24,10 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
    * @param {Address} masterAddress Address of the master safe owning the brackets
    * @param {Transaction} transaction The transaction to be signed and sent
    */
-  const signAndSend = async function (masterSafe, transaction, network) {
-    const nonce = await masterSafe.nonce()
+  const signAndSend = async function (masterSafe, transaction, network, nonce = null) {
+    if (nonce === null) {
+      nonce = (await masterSafe.nonce()).toNumber()
+    }
```

Comment:
If we are already doing this here, why is it being given as a parameter in the function call? is there any reason why we might want to send our own nonce?

Reply:
>we might want to send our own nonce
Yes, in this PR we want to send the second transaction to the interface before the first one was executed. Hence,
`(await masterSafe.nonce()).toNumber()` would give us the wrong nonce for the second transaction. That is why I am calculating the value.